src/commands/import-resolver.ts

```diff
@@ -77,13 +77,18 @@ export function createImportResolver({
       return spec;
     }
     if (dependencyImportMap && dependencyImportMap.imports[spec]) {
+      // if baseURL is remote, handle that outside of path.posix.join()
+      const protocolMatch = config.buildOptions.baseUrl.match(URL_HAS_PROTOCOL_REGEX);
```

Comment:
Didn’t notice the existing RegEx here. Reused what was here, with the bugfix (above)